web/packages/teleport/src/Discover/Shared/SelectCreatable/SelectCreatable.tsx

```diff
@@ -21,7 +21,9 @@ import { Cross } from 'design/Icon';
 
 const styles = theme => ({
   multiValue: (base, state) => {
-    return state.data.isFixed ? { ...base, backgroundColor: 'gray' } : base;
+    return state.data.isFixed
+      ? { ...base, backgroundColor: `${theme.colors.spotBackground[2]}` }
+      : { ...base, backgroundColor: `${theme.colors.spotBackground[1]}` };
```

Comment:
I feel like the difference between fixed and regular is too small. Changing the regular version from spot background 1 to spot background 0 feels better to me, but YMMV.

What's more, if I swap the colors (0 for fixed, 2 for regular), it brings the light version closer to the actual select component. However, the same is not true for the dark version.

I wonder if something like this wouldn't work even better:

```typescript
      backgroundColor: `${
        state.data.isFixed
          ? theme.colors.action.disabledBackground
          : theme.colors.buttons.secondary.default
      }`,
```

The downside is that in the light version, it makes the fixed labels lose the background altogether, though it's not bad for me.

Again, those are just suggestions, if I had to choose I'd compare that against the designs or something.

Reply:
I think going with `spotBackground[0]` instead of `spotBackground[1]` for regular as you suggested looks the best. 
![image](https://github.com/gravitational/teleport/assets/56373201/99361fc0-a8ba-43ac-a8cd-2495a86ec4cd)
![image](https://github.com/gravitational/teleport/assets/56373201/0798e7f5-b7c1-4551-8eba-0e89ff17fd3a)